### DISCUSSION TITLE
How do I disable the widget in chatwoot on mobile devices?

### DISCUSSION THREAD

**Discussion by Yramgar** [chatwoot/chatwoot#7519] [Q&A]
Hello everyone,
I would like to restrict access to the widget I inserted on my website, is it possible to restrict it from loading on mobile devices?
Is it possible to do it through the js code (the code used to insert the widget in my website?

Thank you, I remain attentive.

**Comment by tbraxa** [comment]
How to display the widget on mobile on the left side, and on desktop on the right side? Is that possible?
